apiserver/plane/app/views/issue/base.py

```
@@ -56,7 +56,7 @@
     issue_on_results,
     issue_queryset_grouper,
 )
-from plane.utils.issue_filters import issue_filters
+from plane.utils.issue_filters import issue_filters, build_custom_property_q_objects
```

Pylint (3.3.7)

[error] Unable to import 'plane.utils.issue_filters'

(E0401)